src/server/game/AI/CoreAI/UnitAI.cpp

```diff
@@ -36,7 +36,8 @@ void UnitAI::AttackStart(Unit* victim)
             me->ClearUnitState(UNIT_STATE_DISTRACTED);
             me->GetMotionMaster()->Clear();
         }
-        me->GetMotionMaster()->MoveChase(victim);
+        if (!me->IsControlledByPlayer())
+            me->GetMotionMaster()->MoveChase(victim);
```

Comment:
I'll point this again, as maybe u didnt see my second comment:

There are similar functions that set movement on attack, like AttackStartCaster